examples/datastreams/main/src/example_task.c

```diff
@@ -91,15 +91,13 @@ void astarte_example_task(void *ctx)
         ESP_LOGE(TAG, "Failed to initialize credentials");
         return;
     }
-    /*
-     * For additional ways to define the hwid of your device, see the documentation for
-     * the astarte_device_init function in astarte_device.h
-     *
-     */
```

Comment:
Why did you remove this comment?

Reply:
Seemed very specific regarding the HWID. It also does not add useful additional information. 
It's kind of implied to check the related header for more information regarding a function.
Also, it could be better to encourage users to read the generated Doxygen instead of the header.